src/keymon/keymon.c

```diff
@@ -470,6 +476,22 @@ int main(void) {
                     break;
             }
 
+            if ((val == PRESSED) && (system_state == MODE_MAIN_UI)){
+             // Check for Konami code    
+                if (ev.code == KONAMI_CODE[konamiCodeIndex]) {
+                    ++konamiCodeIndex;
+                    if (konamiCodeIndex == KONAMI_CODE_LENGTH) {
+                        // The entire Konami code was entered!
+                        system("touch /mnt/SDCARD/.tmp_update/config/.konami");
```

Comment:
I think we could make this cleaner by using `cmd_to_run.sh` instead of a flag.
(Then we wouldn't need any edits to `runtime.sh`)

Reply:
We could put this instead:
```c
FILE *file = fopen("/mnt/SDCARD/.tmp_update/cmd_to_run.sh", "w");
fputs("cd /mnt/SDCARD/.tmp_update/bin; ./easter", file);
fclose(file);
```